inspirehep/modules/literaturesuggest/tasks.py

```diff
@@ -153,6 +169,37 @@ def _is_arxiv_url(url):
             date=form_fields.get('thesis_date')
         )
 
+    if form_fields.get('type_of_doc') == 'book chapter':
```

Comment:
Since the logic gets a bit complicated with the addition of these types of documents, add some unit tests (see https://github.com/inspirehep/inspire-next/blob/master/tests/unit/literaturesuggest/test_literaturesuggest_tasks.py#L321) where you isolate different combinations of `type_of_doc` and what you expect to get as the final record.